app/services/search_service.rb

```diff
@@ -149,9 +151,9 @@ def agg_buckets_for_field(field:, current_site: nil, is_crowd_agg: false, url:ni
     options[:smart_aggs] = true
     options[:load] = false
 
-    page = params[:page] || 0
-    page_size = params[:page_size] || DEFAULT_PAGE_SIZE
-    bucket_sort = params[:agg_options_sort] || []
+    page = params.fetch(:page, 0)
```

Comment:
most of this is around the fact that `params` can be nil if we're asking for search data in the context of a nil hash. this is good defensive coding, but it doesn't really solve the problem of assuming we're in the context of a search when we're not.